docs/output.md

```diff
@@ -37,7 +98,103 @@ The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes d
+
+- `preprocessing/adapters/`
+  - `*_overrepresented.fasta`: Contains overrepresented sequences found by FastQC
+
+</details>
+
+[FastQC](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) finds overrepresented sequences found in samples. It lists all of the sequence which make up more than 0.1% of the total. For each overrepresented sequence the program will look for matches in a database of common contaminants and will report the best hit it finds. Hits must be at least 20bp in length and have no more than 1 mismatch.
+
+### Cutadapt
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `preprocessing/cutadapt/`
+  - `*.cutadapt.log`: Cutadapt log file
+  - `*.trim.fastq.gz`: Sample reads trimmed with overrepresented sequences removed
+
+</details>
+
+### Seqtk
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `preprocessing/seqtk/`
+  - `*.seqtk-seq.fastq.gz`: Quality filtered reads.
+
+</details>
+
+[Seqtk](https://github.com/lh3/seqtk) masks (converts to Ns) bases with quality lower than 20 and removes sequences shorter than 80 bases.
+
+<!-- ### UMI -->
+
+## Mapping
+
+### minimap2
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `minimap2/`
+  - `*.bam`: BAM file containing aligned reads
+  - `*.bai`: BAI index
+
+</details>
+
+[Minimap2](https://github.com/lh3/minimap2) is a sequence alignment program that aligns DNA sequences against a reference database.
+
+### BWA
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `bwa/`
+  - `*.bam`: BAM file containing aligned reads
+  - `*.bai`: BAI index
+
+</details>
+
+[BWA-MEM](https://github.com/lh3/bwa) BWA is a software package for mapping low-divergent sequences against a reference genome.
+
+### bowtie2
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `bowtie2/`
+  - `*.bam`: BAM file containing aligned reads
+  - `*.bai`: BAI index
+
+</details>
+
+[Bowtie2](https://bowtie-bio.sourceforge.net/bowtie2/index.shtml) aligns sequencing reads to reference sequences.
+
+## Edits calling
+
+### CIGAR
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `cigar/`
+  - `*_cutSite.json`: Contains the protospacer cut site position in the reference.
+  - `*_edition.html`: Interactive pie chart with the percentage of edition types. Reads are classified between WT (without an edit) and indels. Indes are divided between deletions, insertions and delins (deletion + insertion). Deletions and insertions can be out of frame or in frame.
+  - `*_edits.csv`: Table containing the data for the edition pie chart.
+  - `*_indels.csv`: Table containing information of all reads. Edit tipe, edit start and length, if the edition happens above the error rate, if it's located into the common eddit window, the frequence, the percentage, the pattern, surrounding nucleotides in case of insertions, the protospacer cut site, the sample id, number of aligned reads and number of reads with and without a template modification.
```

Comment:
```suggestion
  - `*_indels.csv`: Table containing information of all reads. Edit type, edit start and length, if the edition happens above the error rate, if it's located into the common edit window, the frequency, the percentage, the pattern, surrounding nucleotides in case of insertions, the protospacer cut site, the sample id, number of aligned reads and number of reads with and without a template modification.
```